packages/tools/addon/styles/addon.css

```diff
@@ -24,43 +26,39 @@
     --faded-tools-highlight: #032c5b77;
 
     --tools-preview: #860404;
-}
 
-@font-face {
-    font-family: 'Avenir';
-    src: url('../@cardstack/tools/Avenir.ttc');
-}
+    /* New Color Scheme */
+    --dark-bg: #01002C;
+    --normal-bg: #343356;
+    --active-bg: #252440;
+    --light-bg: #fff;
+    --gradient-bg: linear-gradient(40deg, #343356 50%, #793284);
+
+    --active-fg: #00e1f9;
+    --bright-fg: #fff;
+    --normal-fg: #000;
+    --purple-fg: #343356;
+    --green-fg: #6aca25;
+    --hot-pink-fg: #FF0054;
+    --gradient-fg: linear-gradient(135deg, #3023ae, #c86dd7);
 
-.cardstack-tools-launcher {
-    background-color: black;
-    cursor: pointer;
+    --light50: rgba(255,255,255,0.5);
+    --light25: rgba(255,255,255,0.25);
+    --light05: rgba(255,255,255,0.05);
+    --dark25: rgba(0,0,0,0.25);
+    --dark50: rgba(0,0,0,0.5);
 }
 
-.cardstack-tools-launcher-wrapper {
-  position: absolute;
-  top: 0;
-  right: 0;
-  z-index: 100;
+
+@font-face {
+    font-family: 'Open Sans';
```

Comment:
I see, that makes sense. I was debating whether to import the bold version, since it didn't look bad but I had Open Sans already installed. I'll import that one too.